sql/driver/conn.go

```diff
@@ -141,6 +150,48 @@ func (c *conn) query(stmt parser.Statement, args []driver.Value) (*rows, error)
 	return nil, fmt.Errorf("TODO(pmattis): unimplemented: %T %s", stmt, stmt)
 }
 
+// Send sends the call to the server.
+func (c *conn) Send(call sqlwire.Call) (*rows, error) {
+	c.sender.Send(context.TODO(), call)
+	resp := call.Reply
+	if resp.Error != nil {
+		return nil, errors.New(resp.Error.Error())
+	}
+	// Translate into rows
+	r := &rows{}
+	// Only use the last result to populate the response
+	index := len(resp.Results) - 1
+	if index < 0 {
+		return r, nil
+	}
+	result := resp.Results[index]
+	r.columns = make([]string, len(result.Columns))
+	for i, column := range result.Columns {
+		r.columns[i] = column
+	}
+	r.rows = make([]row, len(result.Rows))
+	for i, p := range result.Rows {
+		t := make(row, len(p.Values))
+		for j, datum := range p.Values {
+			if datum.BoolVal != nil {
+				t[j] = *datum.BoolVal
+			} else if datum.IntVal != nil {
+				t[j] = *datum.IntVal
+			} else if datum.UintVal != nil {
+				t[j] = *datum.UintVal
```

Comment:
FYI, I think `driver.Value` is not intended to allow `uint64`. Should probably call `database/sql/driver.IsScanValue` on the result to verify we're doing this correctly.